powerhome/playbook
# Improvement to Dialog Targeting

#### Screens

#### Breaking Changes

No

#### Runway Ticket URL


#### How to test this

Set an id for the dialog kit then inspect the modal to see that id appear on it.


#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
